web/src/app/pages/create-survey/loi-permissions/loi-permissions.component.html

```diff
@@ -14,10 +14,10 @@
   limitations under the License.
 -->
 
-<h1 class="title">When are data collection sites defined?</h1>
+<h1 class="title">How are data collection sites defined?</h1>
 <div class="description">
-  Specify whether data is to be collected for predefined locations and/or
-  collected opportunistically as locations are encountered.
+  Is data to be collected for a predefined set of data collection sites, for sites
```

Comment:
The sentence seems a bit repetitive, maybe 

"Is data to be collected for predefined sites, for sites defined by data collectors as they're encountered, or both?